test_cases/regression_tests/jira_142.py

```diff
@@ -0,0 +1,20 @@
+from algopy import ARC4Contract, arc4, log, subroutine
+
+
+class MyStruct(arc4.Struct):
+    foo: arc4.UInt64
+
+
+class AContract(ARC4Contract):
```

Comment:
If you name this MyContract it'll produce a shorter entry in `sizes.txt`... lol
Which, looks like that hasn't been updated yet, so CI will fail in a few more minutes.

Reply:
`sizes.txt` has been updated. :)